sources/packages/web/src/services/http/dto/ApplicationRestrictionBypass.dto.ts

```diff
@@ -0,0 +1,33 @@
+import { RestrictionType } from "@/types";
+
+/**
+ * DTO class for AEST application restriction bypass summary.
+ */
+export class ApplicationRestrictionBypassSummary {
```

Comment:
The DTOs in web are simple interfaces. We use the class for DTO in APIs to adapt with swagger and class validators. For Web we do not need that.

Reply:
Updated.